### ISSUE TITLE
error settings escrow passphrase

### ISSUE THREAD

**Issue by davidkast** [square/luks2crypt#38]
Hello,
after generating the file luks2crypt and executing it
`**./luks2crypt postimaging --luksdevice "eureddesk2" --currentpassword "mi_contrasena" --cryptserver "172.20.46.101"**`
I get the following error:
**error settings escrow passphrase: crypt_init returned error with code -15**
[http://floomby.io/s/030719/EnlYxRM0](URL)

The test has been relied on with Ubuntu Desktop 16.04 encrypting the Operating System since installation.

Can you help me?

Thank you.

**Comment by derektamsen** [comment]
Hi @davidkast, when crypt_init returns an error code the issue is typically with the encrypted luks device. Try using the encrypted formatted partition full device path for the flag `--luksdevice "/dev/some_disk_partition"`. Typically this is the root device like `/dev/sda3`.

You can check which device it might be with `sudo lsblk -f` and look for the root device. This will be the `/dev/sdXN` and is usually of FSTYPE `crypto_LUKS` and in the tree for mountpoint `/`

**Comment by davidkast** [comment]
Hello,
thanks for answering, I saw that I had to add /dev/sda5
and now the following message appears:

error setting escrow passphrase: Pots https://myservercrypt/checkin/: x509: certificate signed by unknown authority.

The server has a CA certificate from the Windows DC, in the linux client it is not in domain and it does not have a certificate, it could be the problem?

Thank you.

[image](http://floomby.io/s/040719/MMJTU3Fc)
